manim/mobject/value_tracker.py

```diff
@@ -57,6 +57,9 @@ def set_value(self, value):
     def increment_value(self, d_value):
         self.set_value(self.get_value() + d_value)
 
+    def __iadd__(self, d_value):
+        return ValueTracker(self.get_value() + d_value)
```

Comment:
I like this approach, but it creates a new instance of `ValueTracker` instead of mutating the existing one. This has two problems:
- references to the old object won't see the change
- it's inefficient to create a new object

It might be better to instead use the `increment_value` method, and then return `self`.

Reply:
Thanks for the review! That is exactly what I was trying to figure out (not creating a new `ValueTracker`). Fixed in the next commit.